src/olympia/abuse/models.py

```diff
@@ -638,6 +639,13 @@ class AbuseReport(ModelBase):
         null=True,
         help_text='Type of illegal content',
     )
+    illegal_subcategory = models.PositiveSmallIntegerField(
+        default=None,
+        choices=ILLEGAL_SUBCATEGORIES.choices,
+        blank=True,
+        null=True,
+        help_text='Specific violation',
```

Comment:
```suggestion
        help_text='Specific violation of illegal content',
```
or some better wording.  Just to make it clear it's connected to the `illegal_category`